### ISSUE TITLE
Problem with inference using pre-trained models

### ISSUE THREAD

**Issue by nanase-123** [IMSY-DKFZ/self-distilled-swin#9]
Hello,I meet a problem when I  use  pre-trained models to inference. I use command 'python generate.py inference=true pretrained_model=true save_folder=official_results exp=SwinT' ,the steps to download the pre-trained model and prepare the data are correct, but there is no follow-up reaction or prompt after starting the inference.
![20241101160535](https://github.com/user-attachments/assets/4dc5995f-33c3-47ed-bf5d-cfd8fd90c09a)
Please help to analyze the reasons for the above problems, I would appreciate it very much

**Comment by nanase-123** [comment]
In addition, I meet a similar problem when training the model, generating soft labels with no reaction or prompt for a long time.
![20241101161331](https://github.com/user-attachments/assets/2713ca5c-f9f4-4d24-8b66-f770e42aeb32)

**Comment by Amiiney** [comment]
Hi @nanase-123, thanks for your feedback! I tried to rerun both training and inference and it works fine. Usually inference and soft-labels generation take a few minutes to run, if you monitor the GPU utilization, it should be high if the code is running. Could you please confirm if it's working?
